mrt/manager.py

```diff
@@ -210,3 +209,26 @@ def remove_missing_countries(ctx):
                 participant.represented_country = None
 
         db.session.commit()
+
+@cli.command()
+@click.pass_context
+def add_verified_flag_mp(ctx):
+    app = ctx.obj['app']
+    with app.test_request_context():
+        for meeting in Meeting.query.all():
+            if meeting.settings.get('media_participant_enabled', False):
+                if meeting.custom_fields.filter_by(
+                            custom_field_type = unicode(CustomField.MEDIA),
+                            slug = 'verified').all() != []:
```

Comment:
That is not how you check if a list is empty in Python. There are some objects, like `None`, empty sequences, zero (extensive list [here](https://docs.python.org/2.4/lib/truth.html)), that evaluate as `False`. Any other object is `True`. So you can just check for a non-empty list like this:
```
if my_list: 
    do_something(my_list)
```

In this particular case, it's better to call `.count()` on that query, which will return `0` for an empty list, to avoid fetching all the data.